archetypes/database-mp/src/test/resources/projects/it1/goal.txt

```diff
@@ -0,0 +1 @@
+package
```

Comment:
I don't know what this file does; if it's OK then…good and carry on!

Reply:
That's left out of the archetype after generation.